### PR TITLE
LPD-43309 Translate tooltip in fieldset taglib

### PR DESCRIPTION
**Pull request by fortunatomaldonado**
https://liferay.atlassian.net/browse/LPP-56950
https://liferay.atlassian.net/browse/LPD-43309

Added translations to the aria-label and title so the tooltip would function correctly.

The playwright test checks to see if the tooltip is hidden and translated correctly.
Please let me know if there are any questions or comments about this.

Thank you!

### CODE REVIEW

**Comment by fortunatomaldonado** [comment]
ci:test:sf

**Comment by fortunatomaldonado** [comment]
ci:test:sf

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

**Comment by fortunatomaldonado** [comment]
ci:test:sf

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

File: modules/test/playwright/tests/frontend-taglib/fieldsetTaglib.spec.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+				});
+
+			await journalPage.goto(site.friendlyUrlPath);
+
+			await journalPage.changeView('cards');
+
+			await page
+				.locator(`.card-body:has-text('${webContentFolder.name}')`)
+				.getByLabel('More actions')
+				.click();
```
Comment (markocikos) [inline modules/test/playwright/tests/frontend-taglib/fieldsetTaglib.spec.ts:39]: As it is, this test should be in `journal-web`, and sent for review to the product team. It uses a lot of web content specifics, that product team can change at any time.

Alternative is to add a sample of tag in `frontend-taglib-sample-web`, and add a test in something like
```
/frontend-taglib/tests/liferay-frontend/fieldset.spec.ts
```

File: modules/test/playwright/tests/frontend-taglib/fieldsetTaglib.spec.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+			await page.getByRole('menuitem', {name: 'Edit'}).click();
+		});
+
+		await test.step('Edit Web Content Folder and check tooltip', async () => {
+			const svgElement = page.locator(
+				'svg[aria-label="Folders can be restricted to only allow certain structures."]'
+			);
+
+			await expect(svgElement).toBeHidden();
```
Comment (markocikos) [inline modules/test/playwright/tests/frontend-taglib/fieldsetTaglib.spec.ts:49]: `toBeAttached()` is probably enough.

**Review by markocikos — CHANGES_REQUESTED** [review]
@fortunatomaldonado The solution is fine, but the test should be moved or changed

File: modules/test/playwright/tests/frontend-taglib/fieldsetTaglib.spec.ts (hunk 1 above)
Comment (fortunatomaldonado) [reply modules/test/playwright/tests/frontend-taglib/fieldsetTaglib.spec.ts]: Thank you, I will be making changes to journal-web and sending to product team!

**Comment by fortunatomaldonado** [comment]
FWD: https://github.com/liferay-content-management/liferay-portal/pull/6159

**Comment by fortunatomaldonado** [comment]
Hello @markocikos 

According to Ambrin here, https://github.com/liferay-content-management/liferay-portal/pull/6159#pullrequestreview-2488228758, they are trying to avoid tests for small visual changes like the tooltip.

She suggests to remove the tests and continue with the pull as is. 
What do you believe we should do with the test?

Thank you!

**Comment by markocikos** [comment]
@fortunatomaldonado As we cannot forward PR without a test, our only option here is to add a sample to `frontend-taglib-sample-web`, and write test on it.

**Comment by fortunatomaldonado** [comment]
ci:test:sf

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

File: modules/test/playwright/tests/frontend-taglib/frontend-taglib-sample-web/fieldset.spec.ts
Comment (markocikos) [inline modules/test/playwright/tests/frontend-taglib/frontend-taglib-sample-web/fieldset.spec.ts:1]: @fortunatomaldonado The code and test are fine, just the placement of file is off. See [related slack discussion](https://liferay.slack.com/archives/C3JBR21HA/p1733916467636509).

Based on current conclusions in the thread, this should be 
`modules/test/playwright/tests/frontend-taglib/tests/frontend-taglib/fieldset.spec.ts`

In the slack thread, there are two other things that I'm suggesting:
1. moving `Add taglib sample to page` to a page property util
2. making a two-layer navigation

These two are not blockers for this PR.

Comment (fortunatomaldonado) [reply modules/test/playwright/tests/frontend-taglib/frontend-taglib-sample-web/fieldset.spec.ts:1]: @markocikos, made the file changes!

Let me know if theres anything else needed.

**Comment by fortunatomaldonado** [comment]
ci:test:sf

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

**Review by markocikos — APPROVED** [review]
@fortunatomaldonado LGTM! :sparkles:

**Comment by fortunatomaldonado** [comment]
ci:forward

**Pull request closed without merging**